templates/zerver/archive/home.html

```diff
@@ -0,0 +1,15 @@
+{% if is_web_public and message_list %}
+<div class="message_table focused_table" id="zhome">
+    {% for msg in message_list %}
+    {{ msg|safe }}
+    {% endfor %}
+</div>
+{% elif is_web_public %}
+<p>
+    This topic does not exist.
+</p>
+{% else %}
+<p>
+    This stream does not exist.
```

Comment:
I think we should make this error message clear about the possibilities: "This stream either does not exist or is not publicly archived to the web.".

Reply:
(And we should tag these strings for translation).